testbed/addons/flux/construct.ts

```diff
@@ -0,0 +1,85 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+import { KubernetesManifest } from '@aws-cdk/aws-eks';
+
+export interface FluxProps extends cdk.StackProps {
+    cluster: eks.Cluster;
+    repositories: Repository[];
+}
+
+export interface RepositoryOptions {
+    branch?: string;
+    path?: string;
+    url: string;
+    name: string;
+}
+
+export class Repository {
```

Comment:
I wonder if would simplify things to just remove this class entirely and instead use solely the interface. The consumer (flux construct) can do the ?? defaulting inline.